start

```diff
@@ -112,6 +110,13 @@ nix_toolchain=$(cat <<EOF
 load(
     "@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
     "nixpkgs_git_repository",
+    "nixpkgs_python_configure",
+)
+
+nixpkgs_python_configure(
+    # XXX: Remove python2_attribute_path after updating to Stardoc 0.4.0
+    python2_attribute_path = "python2",
```

Comment:
Those two lines can be removed. As the comment indicates, they're only needed for the docs generation in `rules_haskell`.